app/components/bookings/create/create-booking.component.ts

```diff
@@ -1,9 +1,124 @@
+      '_dateControl': ['', Validators.required],
+      '_effortControl': ['', Validators.required]
+    });
+
+    this._newEtcControl = this._authForm.controls['_newEtcControl'];
+    this._effortControl = this._authForm.controls['_effortControl'];
+    this._descriptionControl = this._authForm.controls['_descriptionControl'];
+    this._dateControl = this._authForm.controls['_dateControl'];
+    this._Etc = this._workpackage.etc;
+  }
+
+  onSubmit() {
+    if (this._checkFieldsSet()) {
+      var booking: Booking = new Booking();
+      booking.date = moment(this._dateControl.value);
+      booking.description = this._descriptionControl.value;
+      booking.effort = this._stringToWorkday(this._effortControl.value);
+      booking.workpackage = this._workpackage;
+      this._bookingsService.create(booking, this._Etc).subscribe((booking: Booking) => {
+        if(this._bookingSession) {
+          //if there is a bookingsession, delete it when the booking was successful
+          this._bookingSessionService.delete(this._bookingSession).subscribe((response: Response) => {
+
+          });
+        }
+        //TODO: show ToastMsg Booking successful
+        this._navigateToBookingOverview();
+      },
+        error => {
+          let alert = Alert.create({
+            title: 'Buchung fehlgeschlagen!',
+            subTitle: 'Verbindung zum Server konnte nicht hergestellt werden.',
+            buttons: ['OK']
+          });
+          this._navController.present(alert);
+        }
+      );
+    }
+  }
+
+  createBookingSession(): void {
+    this._bookingSession = new BookingSession();
+    this._effort = moment().hours(0).minutes(0).format('HH:mm');
+    this._bookingSession.workpackage = this._workpackage;
+    this._bookingSessionService.create(this._bookingSession).subscribe((bookingSession: BookingSession) => {
+      this._bookingSession = bookingSession;
+    });
+  }
+
+  refreshEtc(): void {
+    this._Etc = this._workpackage.etc - this._stringToWorkday(this._effortControl.value);
+  }
+
+  private _checkFieldsSet(): boolean {
+    return this._authForm.value._descriptionControl != '';
+  }
+
+  private _stringToWorkday(_effortControl: string): number {
+    let timeStrings = _effortControl.split(':');
+    let hoursPerDay = parseInt(timeStrings[0])/8;
+    let minutesPerDay = (parseInt(timeStrings[1])/60) /8;
+    return hoursPerDay+minutesPerDay;
+  }
+
+  private _calcEffortForSession(start: number, end: number): string {
+    let startMoment = moment(start);
+    let endMoment = moment(end);
+    let hours = endMoment.diff(startMoment, 'hours');
+    let minutes = (endMoment.diff(startMoment, 'minutes'))%60;
+    let effortMoment = moment().hours(hours).minutes(minutes);
+    return effortMoment.format('HH mm');
+  }
+
+  private _navigateToBookingOverview(): void {
+    this._navController.push(BookingOverviewComponent, {
+      bookingSession: this._bookingSession
```

Comment:
Can the `BookingOverviewComponent` take a `BookingSession`? 
I don't think this would be a good mechanic. If this was done, to prevent the `BookingOverviewComponent` from having to do another network request, I think a better approach would be to cache the current `BookingSession` in the `BookingSessionService`, instead of keep reaching it forward to anyone who needs it. But I think we can forgo this, in simplicities favor.